backend/src/models/delete.model.ts

```diff
@@ -0,0 +1,4 @@
+export interface DeleteRequest {
+    deleterId: number;
+    userToDeleteId: number;
+}
```

Comment:
If you use the approach from above, you don't need a separate model 
the frontend will simply send a delete request and the id of the user which should be deleted.